tslint.json

```diff
@@ -1,35 +1,42 @@
 {
```

Comment:
The only change to the file is the addition of `"member-access": true,`, and then I ran reformat...